.github/workflows/note-arduino-ci.yml

```
@@ -0,0 +1,71 @@
+name: Note Tutorials CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  validate_esp32_examples: # job id
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        include:
+          - sketch: sensor-tutorials/arduino-wiring/nano-no-library
+            fully-qualified-board-name: esp32:esp32:nano32
```

I think this is supposed to be the Arduino variant
```suggestion
            fully-qualified-board-name: arduino:mbed_nano:nano33ble
```